### ISSUE TITLE
Collect Initial binrw Community Showcase

### ISSUE THREAD

**Issue by jam1garner** [jam1garner/binrw#90]
TLDR: post your projects using binrw/binread below

* [x] Pull showcases from discord
* [x] Pull showcases from "dependents" on crates.io
* [x] Ask around for additional showcase nominations

Rules:
* Projects using binread will be counted
* Any crypto or crypto-adjacent projects will not be included
* All projects must have at least one maintainer/primary author agree to inclusion

Things to include:
* Screenshot (if possible)
* 1-sentence summary
* Link to docs.rs, crates.io, github, or other relevant site (multiple is acceptable)
* Preferred name(s) to list as the author(s) (Optional)
  * Pronouns may also 100% be included if you so choose. I will personally include mine so as to not make anyone feel they are singling themselves out by doing so.

Self-nomination is both accepted and highly encouraged! Some of the coolest projects don't have more than 3 pairs of eyes on them. Showcased projects can be removed at any point on request.

**Comment by leo60228** [comment]
* Goethite is an experimental from-scratch Pebble smartwatch SDK written in Rust ~~that I'm definitely going to start working on again soon I swear~~, which uses binrw for handling the custom binary format.
* https://gitlab.com/goethite

**Comment by octylFractal** [comment]
https://github.com/octylFractal/choadra minecraft protocol shenanigans but very incomplete

**Comment by ScanMountGoat** [comment]
- ssbh_lib is a library for reading and writing binary file formats related to rendering for Super Smash Bros Ultimate and some other games.
- https://crates.io/crates/ssbh_lib

**Comment by vivlim** [comment]
* dkpunchbot is a twitter bot which records videos of Donkey Kong throwing extremely strong punches. binrw is used to read and write N64 emulator RAM for choreography and to adjust various attributes including the strength of DK's punch.
* https://twitter.com/dkpunchbot/
* written by viv (she/her), vivviridian on twitter

I don't have any shareable code atm but will try to remember to let you know when I do.

**Comment by jam1garner** [comment]
Still accepting submissions, but going to close the issue as the initial showcase is made. More can still be added/updated whenever
